src/main/resources/static/css/login.css

```diff
@@ -0,0 +1,71 @@
+@media (min-width: 640px) {
+        .label-container {font-size: 24px;}
+        .input-field {min-width: 220px;}
+        .form-container {gap: 1.25rem;}
+        .invalid-state {font-size: 14px;}
+        .warning-state {font-size: 14px;}
+        .login-button {font-size: 20px;}
+	}
+
+@media (max-width: 640px) {
+        .label-container {font-size: 14px;}
+        .input-field {min-width: 150px;}
+        .form-container {gap: 0.75rem;}
+        .invalid-state {font-size: 10px;}
+        .warning-state {font-size: 10px;}
+        .login-button {font-size: 14px;}
+    }
+
+.login-container {
+    box-sizing: border-box;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.form-container {
+    display: inline-flex;
+    flex-direction: column;
+}
+
+.label-container {
+    display: inline-flex;
+    flex-direction: column;
+}
+
+.login-button {
+    margin-left: auto;
+}
+
+.login-button > input {
+    font-weight: 400;
+	text-align: center;
+	border: 1px solid transparent;
```

Comment:
A transparent border sort of does the same thing as extra padding. Why do you have a transparent border and not just add a bit more padding?

Reply:
Hello @TimLeach635 ,

The reason was to remove the default border for input fields. Without it, it looks like this:
![Screenshot from 2023-10-11 11-06-10](https://github.com/charlottemothersole/WhaleSpotting-Java/assets/114360842/bd80e8f2-dfc9-403d-91af-e1daf3035608)

And with the transparent border:
![Screenshot from 2023-10-11 11-01-50](https://github.com/charlottemothersole/WhaleSpotting-Java/assets/114360842/4a8036cf-6971-429a-9da1-85d547a8643b)

Reply:
I'll change it now then!